.github/workflows/tests.yml

```diff
@@ -1,5 +1,5 @@
 ---
-name: Unit tests
```

Comment:
Aren't these still unit tests?

Reply:
1. Some tests go beyond what I would call a unit test.
F.e. (and it is not a good example): `TestNVMEIsUsed` is not isolated from the system, because it opens `/sys/bus/pci/devices/0000:00:00.0/nvme`.
2. I think we should also allow having tests that are beyond unit tests.

Reply:
While that might be the case, having less specific names doesn't make things more clear, since that might create more confusion between this and the eden tests.

So if you want to call it "unit and xyzzy tests" that might be more clear (but I don't know what value of xyzzy would provide clarity on the scope.)

Reply:
Renamed to "Go Tests"